### ISSUE TITLE
Feature Request: Store Locator

### ISSUE THREAD

**Issue by craigscott29** [ampproject/amphtml#17485]
On the website I manage, we have a [dealer/store locator](https://www.craigattachments.com/find-a-dealer/) for our customers. This page is the only page on our site that is not AMP compliant due to all the JavaScript required to run that feature. It's currently a WordPress plug-in, but I'm working on building a custom one to tie into our ERP API. Through doing this it got me thinking, is something like this possible within the scope of AMP?

![find-a-dealer](https://user-images.githubusercontent.com/33098508/44095141-7119d3f0-9fae-11e8-91a2-2f6ca8c7de37.PNG)

I am aware of the [AMP geolocation capabilities](https://ampbyexample.com/advanced/geolocation_with_amp-list/) and the ability to build out static maps through amp-list. The features this appears to lack are the ability to add multiple markers on a single interactive map and the ability to search within a radius of the users inputted location.

Basically, I'm looking for some sort of combination of amp-list and amp-bind that integrates with a map and allows for:

- User inputted address searching
- [Radius filtering](https://en.wikipedia.org/wiki/Haversine_formula)
- Custom attributes filtering
- Multiple markers on an interactive map

Perhaps something like this is possible now and I'm just not thinking/looking hard enough. If that's the case, please point me in the direction to go searching.

**Comment by aghassemi** [comment]
@craigscott29 For now, `amp-iframe` is the only way. We are considering adding `amp-maps` component (#12209) but that's currently not prioritized.

**Comment by aghassemi** [comment]
closing per comment above.
